GliaWidgets/Configuration.swift

```diff
@@ -1,6 +1,10 @@
+/// Glia's environment.
 public enum Environment {
```

Comment:
This should be expanded, with something like "Use the one that our account manager has assigned to you."